components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts

```diff
@@ -1,11 +1,16 @@
 import { HttpClient, HttpParams } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { reduce } from 'lodash/fp';
 import { Injectable } from '@angular/core';
 
 import {
-  ServiceGroupsPayload, ServiceGroupFilters,
+  ServiceGroupsPayload,
+  ServiceGroupFilters,
+  ServiceGroupSuggestion,
   HealthSummary,
-  ServicesPayload, ServicesFilters
+  ServicesPayload,
+  ServicesFilters
 } from './service-groups.model';
 import { environment } from '../../../environments/environment';
```

Comment:
Another instance of that path convention I mentioned above. 😁 
```suggestion
import { environment } from 'environments/environment';
```